Util/AssetPlugin.php

```diff
@@ -100,4 +101,28 @@ public static function setVcsTypeRepositories(RepositoryManager $rm)
             }
         }
     }
+
+    /**
+     * Adds the main file definitions from the root package.
+     *
+     * @param Composer         $composer
+     * @param PackageInterface $package
+     * @param string           $section
+     */
+    public static function addMainFiles(Composer $composer, PackageInterface $package, $section = 'asset-main-files')
```

Comment:
Because the `setExtra` method is only in the `Package` class, you must use the `Package` typehint and not the `PackageInterface`.